hypha/locale/django.pot

```diff
@@ -6695,22 +6679,6 @@ msgstr ""
 msgid "You might also like…"
 msgstr ""
 
-#: hypha/templates/includes/share.html:4
-msgid "Share"
-msgstr ""
-
-#: hypha/templates/includes/share.html:8
-msgid "Share on Twitter"
-msgstr ""
-
-#: hypha/templates/includes/share.html:14
-msgid "Share on LinkedIn"
-msgstr ""
-
-#: hypha/templates/includes/share.html:21
-msgid "Share on Facebook"
-msgstr ""
-
```

Comment:
This file is auto generated, better to update in a separate PR.

Reply:
I agree, but can we get this merged and i'll take care of in the future PRs.